Public/Src/Tools/Execution.Analyzer/Analyzers.Core/XLGPlusPlus/Events.proto

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+syntax = "proto3";
+
+package BuildXL.Execution.Analyzer;
+
+option csharp_namespace = "BuildXL.Execution.Analyzer";
+
+message DominoInvocationEvent{
+    string UUID = 1;
+
+    uint32 WorkerID = 2;
+
+    string SubstSource = 3;
+    
+    string SubstTarget = 4;
+
+    bool IsSubstSourceValid = 5;
+    
+    bool IsSubstTargetValid = 6; 
+}
+
+message DominoInvocationEventList{

```

Comment:
>DominoI [](start = 8, length = 7)

I know the exiting events are still all called `DominoInvocation` that is mostly because the Microsoft internal telemetry system has many events and queries on top of the telemetry for this event that it was not worth renaming.

Since this is going to be a new api surface, would you mind updating to BuildXLInvocation for the events?

Note: Not needed before checking in, can be done in a future PR.

Reply:
You're right, I'll change it for this PR. 

---
In reply to: [305538922](https://github.com/microsoft/BuildXL/pull/636#discussion_r305538922) [](ancestors = 305538922)